foundry/game/File.py

```diff
@@ -15,11 +16,32 @@
 TSA_TABLE_SIZE = 0x400
 TSA_TABLE_INTERVAL = TSA_TABLE_SIZE + 0x1C00
 
+class InesHdr(Structure):
+    _fields_ = [
+        ("magic", c_char * 4),
+        ("prg_units", c_ubyte),
+        ("chr_units", c_ubyte),
+        ("flags6", c_char),
+        ("unused_flags", c_char * 4),
+        ("unused_pad", c_char * 5)
+    ]
+    PRG_UNIT_SIZE = 0x4000
+    CHR_UNIT_SIZE = 0x2000
+    def __init__(self, data):
+        self.raw_data = data[0:sizeof(self)]
+        memmove(addressof(self), data, sizeof(self))
+    @property
+    def prg_size(self):
+        return self.prg_units * InesHdr.PRG_UNIT_SIZE
+    @property
+    def chr_size(self):
+        return self.chr_units * InesHdr.CHR_UNIT_SIZE
```

Comment:
Fix lines to follow PEP 8 style guidelines.
```suggestion
    def __init__(self, data):
        self.raw_data = data[0:sizeof(self)]
        memmove(addressof(self), data, sizeof(self))
        
    @property
    def prg_size(self):
        return self.prg_units * InesHdr.PRG_UNIT_SIZE
        
    @property
    def chr_size(self):
        return self.chr_units * InesHdr.CHR_UNIT_SIZE
```